---
# flag to enable/disable debug
labelerDebug: false
analyzerDebug: false

# sampling voxel size
voxelSize: 0.005
# location of the BoW file
bowLocation: 'src/grasping/config/bow200_1E-5.dat'
# radius used for the normal vectors estimation
normalEstimationRadius: -1

##### setup node options #####
setup:
    liftTorso: false
    moveArms: true
    moveBase: false
    moveHead: true

    torsoPosition: 0.19

    displacementThreshold: 1.0

    armsPosition:
      right: {x: 0.3, y: -0.5, z: 1.0}
      left: {x: 0.3, y: 0.5, z: 1.0}

    headTarget: {x: 0.9, y: 0.0, z: 0.5}

##### descriptor calculation params #####
descriptor:
    patchSize: 0.02
    bandNumber: 8
    bandWidth: 0.005
    bidirectional: false
    useProjection: false
    sequenceBin: 0.005
    sequenceStat: mean # mean or median

##### cloud smoothing params #####
cloudSmoothing:
    useSmoothing: true
    sigma: 4
    radius: 0.005
...